comid/measurement.go

```diff
@@ -97,7 +97,7 @@ type Mval struct {
 	SVN          *SVN      `cbor:"1,keyasint,omitempty" json:"svn,omitempty"`
 	Digests      *Digests  `cbor:"2,keyasint,omitempty" json:"digests,omitempty"`
 	OpFlags      *OpFlags  `cbor:"3,keyasint,omitempty" json:"op-flags,omitempty"`
-	RawValue     *[]byte   `cbor:"4,keyasint,omitempty" json:"raw-value,omitempty"`
+	RawValue     *RawValue `cbor:"4,keyasint,omitempty" json:"raw-value,omitempty"`
```

Comment:
Wondering we set this incorrectly. I am little confused, as to why RawValue and RawValueMask is not logically clubbed into a single entity, aligning it with CDDL Schema.

I think, they are logically seen as grouped entity?

Reply:
saying
```
raw-value-group = (
  comid.raw-value => $raw-value-type-choice
  ? comid.raw-value-mask => raw-value-mask-type
 )
```
 and then
```
measurement-values-map = non-empty<{
  [...]
  ? raw-value-group
  [...]
}>
```
is just a different way to say
```
measurement-values-map = non-empty<{
  [...]
  ? ( comid.raw-value => $raw-value-type-choice, ? comid.raw-value-mask => raw-value-mask-type )
  [...]
}>
```